test/integration/document/Controls.e2e.test.js

```diff
@@ -0,0 +1,108 @@
+// <reference types="Cypress" />
+describe('Preview Document Controls', () => {
+    const token = Cypress.env('ACCESS_TOKEN');
+    const fileId = Cypress.env('FILE_ID_DOC');
+
+    /* eslint-disable */
+    const getPage = (pageNum) => cy.get(`.page[data-page-number=${pageNum}]`);
+    const showControls = () => {
+        // Hover over the preview to trigger the controls
+        cy.getByTestId('bp').trigger('mouseover');
+        // Assert that the controls are shown
```

Comment:
I'm a fan of code comments, but this logic seems pretty self-explanatory to me. Is a comment needed here?